src/scenes/home/faq/question/question.js

```diff
@@ -1,19 +1,17 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './question.css';
 
-class Question extends Component {
-  render() {
-    return (
-      <div className={styles.accordionSingle}>
-        <input className={styles.accordionSingleHidden} type="checkbox" id={this.props.q} />
-        <label className={styles.accordionSingleQuestion} htmlFor={this.props.q}> {this.props.q}</label>
-        { // eslint-disable-next-line
-          <p className={styles.accordionSingleAnswer} dangerouslySetInnerHTML={{ __html: this.props.a }} />
-        }
-      </div>
-    );
-  }
+const Question = () => {
```

Comment:
Thanks so much for your work. Travis has been told by our linter that you must do the following instead:

```
const Question = ({q, a}) => (
  <div className={styles.accordionSingle}>
    <input className={styles.accordionSingleHidden} type="checkbox" id={q} />
    <label className={styles.accordionSingleQuestion} htmlFor={q}> {q}</label>
    { // eslint-disable-next-line
      <p className={styles.accordionSingleAnswer} dangerouslySetInnerHTML={{ __html: a }} />
    }
  </div>
);
```

Reply:
O o.. It seems I was in too hurry to do my first #hactoberfest PR. 
I'll just correct it.

Reply:
You've got all the time in the world 👍

Reply:
I messed up no of commits. Let me fix it. and summerise it in two commits.

Reply:
Yay! Now it looks good.